### PR TITLE
[MODEL-11706] simple hyperparam examples

### PR DESCRIPTION
**Pull request by scottp-ml**
# This repository is public. Do not put here any private DataRobot or customer's data: code, datasets, model artifacts, .etc.

## Summary
Add hyperparameter examples where appropriate in custom tasks.

### CODE REVIEW

File: task_templates/2_estimators/1_python_regression/custom.py [hunk 1]
```diff
@@ -13,7 +13,7 @@
 
 
 class CustomTask(RegressionEstimatorInterface):
-    def fit(self, X: pd.DataFrame, y: pd.Series, **kwargs) -> None:
+    def fit(self, X: pd.DataFrame, y: pd.Series, parameters=None, **kwargs) -> None:
```
Comment (akshoop) [inline task_templates/2_estimators/1_python_regression/custom.py:16]: Be sure to update docstring in all changed places to reflect new `parameters` parameter

**Review by akshoop — APPROVED** [review]
LGTM from Core Modeling, CC @eric-s-s for 2nd reviewer

**Comment by akshoop** [comment]
Suggest updating `task_templates` README to mention using `hyperparameters` functionality. Also update any outdated information such as the mention of non-existent `python3_sklearn_transform_hyperparameters` here:
https://github.com/datarobot/datarobot-user-models/tree/master/task_templates#transforms

**Review by eric-s-s — APPROVED** [review]
:100: nice!

**Comment by scottp-ml** [comment]
jarvis retest failing

**Comment by scottp-ml** [comment]
jarvis test test_integration_r_environment

**Pull request merged**